docs/section11/Rule11-12.md

```diff
@@ -0,0 +1,39 @@
+# Service_Water_Heating - Rule 11-12
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** FALSE
+
+**Rule ID:** 11-12  
+
+**Rule Description:** "For large, 24-hour-per-day facilities that meet the prescriptive criteria for use of condenser heat recovery systems described in Section 6.5.6.2, a system meeting the requirements of that section shall be included in the baseline building design regardless of the exceptions to Section 6.5.6.2."  
+
+**Rule Assertion:** UNDETERMINED / NOT_APPLICABLE
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, d + exception
+
+**Evaluation Context:** B_RMD - each building
+**Data Lookup:**   
+**Function Call:** 
+- **get_spaces_associated_with_each_SWH_bat**
+
+**Applicability Checks:**
+- check that there is SWH in the proposed model AND that the building has a 24hr operating schedule
+
+## Applicability Checks:
+- set the hours_this_year to 8760: `hours_this_year = 8760`
+- if the calendary is_leap_year, then we need to add 24 hours to hours_this_year: `if RMD.calendar.is_leap_year: hours_this_year = hours_this_year +24`
+- look at each building in the RMD: `for building in B_RMD:`
+    - set a boolean is_applicable equal to false: `is_applicable = false`
+    - check to see if the building is operational 24hrs/ day, every day of the year.  Start by getting the building_open_schedule: `operation_schedule = building.building_open_schedule`
+    - check to see if the schedule is operating 24hrs per day, all days per year: `if sum(operation_schedule) == hours_this_year:`
+      - only projects with SWH in the proposed model are interpreted as have in SWH loads.  A building must both have SWH loads and a 24hr operating schedule for this rule to be applicable.
+      - look at each space in the building: `for space in P_RMD.building...:`
+          - check to see if there is at least one SWH use associated with the space: `if len(space.service_water_heating_uses) > 0:`
+              - the rule is applicable: `is_applicable = true`
+      
+    
+    ## Rule Assertion:
+    - if the boolean is_applicable is equal to true, the rule is applicable, return UNDETERMINED and a note letting the reviewer know what to check: `if is_applicable: UNDETERMINED raise_message: "This building is a 24hr-facility with Service Water Heating.  If the building meets the prescriptive criteria for use of condenser heat recovery systems described in Section 6.5.6.2, a system meeting the requirements of that section shall be included in the baseline building design regardless of the exceptions to Section 6.5.6.2."`
```

Comment:
Recommend: "This building is a 24hr-facility with service water heating loads.  If the building meets the prescriptive criteria for use of condenser heat recovery systems described in 90.1 Section 6.5.6.2, a system meeting the requirements of that section shall be included in the baseline building design regardless of the exceptions to Section 6.5.6.2."`

Also, I think it would be helpful to include the exception as well for the reviewer in the message.